src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -436,6 +437,37 @@ static LogicalResult RNNShapeInference(T *op) {
   return success();
 }
 
+static void insertConvTransposeSpatialDim(SmallVector<int64_t, 4> *outputDims,
```

Comment:
It would be better to pass outputDims by reference. The semantics of pointers allow for nullptr to be passed in, which is probably not what you want. Changing it to pass by reference would evade such concerns.